src/services/prepareSubmissionData.ts

```diff
@@ -126,3 +126,41 @@ async function uploadAttachments(
 
   return submission
 }
+
+export function getPaymentElementValuePath(
+  elementId: string,
+  elements: FormTypes.FormElement[],
+): string[] {
+  for (const element of elements) {
+    if (element.id === elementId && element.type !== 'page') {
+      return [element.name]
+    }
```

Comment:
Will this cater for the element being in a page?

Reply:
It will now.